sandpack-react/src/presets/Sandpack.tsx

```diff
@@ -197,16 +214,7 @@ export const Sandpack: SandpackInternal = ({
       {...props}
     >
       <SandpackLayout>
-        <SandpackCodeEditor
-          {...codeEditorOptions}
-          style={{
-            height: options.editorHeight, // use the original editor height
-            flexGrow: horizontalSize,
-            flexShrink: horizontalSize,
-            flexBasis: 0,
-            overflow: "hidden",
-          }}
-        />
+        {!rtlLayout && <CodeEditor />}
```

Comment:
Just a open sugestion, I didn't try it: could we achieve it using CSS `order` props?

Reply:
has changed to CSS